game/scripts/npc/abilities/silencer_glaives_of_wisdom.txt

```diff
@@ -12,7 +12,7 @@
     "AbilityUnitTargetTeam"                               "DOTA_UNIT_TARGET_TEAM_ENEMY"
     "AbilityUnitTargetType"                               "DOTA_UNIT_TARGET_HERO | DOTA_UNIT_TARGET_BASIC"
     "SpellImmunityType"                                   "SPELL_IMMUNITY_ENEMIES_NO"
-    "AbilityUnitDamageType"                               "DAMAGE_TYPE_PHYSICAL"
```

Comment:
On that note though, I don't think the change to physical worked. I tested hitting a unit with Stoneskin on and off with a Silencer with max level Glaives and some 400 int, and Stoneskin hardly changed the damage.